llvm/utils/TableGen/IntrinsicEmitter.cpp

```diff
@@ -256,7 +256,7 @@ void IntrinsicEmitter::EmitIntrinsicToOverloadTable(
 static constexpr uint8_t OTable[] = {
   0
   )";
-  for (auto [I, Int] : enumerate(Ints)) {
+  for (const auto [I, Int] : enumerate(Ints)) {
```

Comment:
Given it is just a copy, do we still need `const`?

Reply:
Removed `const`